frontend/packages/client/src/App.sass

```diff
@@ -9,7 +9,7 @@ $red: #B20D0D
 $error-red: #B20D0D
 $yellow: #FBD84D
 $orange: #F4AF4A
-$light-gray: #DCDCDC
+$light-gray: #F9F9F9
```

Comment:
this was only being used for a non-existent selector so I deleted rules and updated this variable for the light gray in latest designs